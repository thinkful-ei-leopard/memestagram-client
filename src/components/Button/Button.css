button{
    margin: 20px;
    display: inline-block;
    border: 2px solid #fbc99d;
    border-radius: 5px;
    padding: 8px 10px;
    color: #fbc99d;
    font-size: 18px;
    font-family: 'Changa One', cursive;
    background-color: #042f4b;
}

