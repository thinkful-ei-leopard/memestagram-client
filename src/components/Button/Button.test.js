import React from 'react';
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'
import Button from './Button'

describe('Button Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <Button />
      </BrowserRouter>, 
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders Button as expected', () => {
    const tree = renderer
      .create(<Button />)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })
})