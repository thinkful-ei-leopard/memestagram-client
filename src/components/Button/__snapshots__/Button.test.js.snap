// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component renders Button as expected 1`] = `
<button
  className="Button"
/>
`;
